static/debcodesearch.css

```diff
@@ -6,6 +6,11 @@
 	text-align: center;
 }
 
+#searchdiv input[type=text] {
+	width: 35px;
```

Comment:
I must be missing something, because your suggested change makes the search box _a lot_ smaller than it is right now. 35px? Is that a typo? Chrome renders the search box as 169 px for me (without your change).